charts/kured/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "1.6.1"
+appVersion: "1.7.0"
 description: A Helm chart for kured
 name: kured
-version: 2.4.0
+version: 2.4.1
```

Comment:
This will be conflicting on rebase. Instead of bumping to 2.4.1, I would bump the version to 2.5.0, as we are moving to a new version of kured in practice.